WordPress/WordPressTest/ReaderReblogActionTests.swift

```diff
@@ -31,25 +31,15 @@ class MockBlogService: BlogService {
     }
 }
 
-class MockPostService: PostService {
-    var draftPostExpectation: XCTestExpectation?
-
-    override func createDraftPost(for blog: Blog) -> Post {
-        draftPostExpectation?.fulfill()
-        return Post(context: self.managedObjectContext)
-    }
-}
```

Comment:
What's better than improving a test double API and its usage? 

Restructuring the code so that it doesn't need a test double in the first place.

Thanks!